app/src/main/java/com/dongyang/android/youdongknowme/ui/view/depart/DepartActivity.kt

```diff
@@ -62,6 +66,18 @@ class DepartActivity : AppCompatActivity(), DepartClickListener {
         }
     }
 
+    private fun spanText(spanTextView: TextView, startIdx: Int, endEdx: Int){
+        val ssb = SpannableStringBuilder(spanTextView.text)
+        ssb.setSpan(
+            ForegroundColorSpan(getColor(R.color.main)),
+            startIdx,
+            endEdx,
+            Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
+        )
+
+        spanTextView.text = ssb
```

Comment:
```suggestion
        SpannableStringBuilder(binding.departSpanText.text).apply {
            setSpan(
                ForegroundColorSpan(getColor(R.color.main)),
                startIdex,
                endEdx,
                Spannable.SPAN_EXCLUSIVE_EXCLUSIVE
            )

            binding.departSpanText.text = this
        }
```

스코프 함수를 이용하면 조금 더 깔끔하게 구현할 수 있을 것 같아요~